cluster/vagrant/util.sh

```diff
@@ -33,13 +33,62 @@ function detect-minions {
 # Verify prereqs on host machine  Also sets exports USING_KUBE_SCRIPTS=true so
 # that our Vagrantfile doesn't error out.
 function verify-prereqs {
-  for x in vagrant VBoxManage; do
+  for x in vagrant; do
     if ! which "$x" >/dev/null; then
       echo "Can't find $x in PATH, please fix and retry."
       exit 1
     fi
   done
 
+  local vagrant_plugins=$(vagrant plugin list | sed '-Ees% .*$%%' '-Ees% +%\n%g')
```

Comment:
This appears to be the culprit for the unwanted garbage output to terminal when starting kube-up.  Please clean that up so users are not confused when a sed fails.